src/content/docs/style-guide/writing-docs/article-templates/branching-docs-guide.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: How to create a branching doc from scratch
+metaDescription: Our branching doc framework is a powerful tool that we can use to dynamically swap content in and out of, depending on a set of choices.
+---
+
+Branching docs are a powerful tool to represent choices and different content to our users. In the past, we might have represented these choices as a table or a set of collapsers. This new doc format is a flexible and extensible tool that we can use to consolidate our installation docs and innovate on how we present information across our site.
+
+## New components [#new-components]
+
+The branching doc introduces some new components. Some of these components are available for use across the docs site, whether it's a branching doc or not.
+
+The new universal components are:
+
+* installation-specific feedback form
+* tab switcher
+
+The branching-specific components are:
+
+* dropdown menus with image-based tile selection
+* numbered steps
+* embedded, customizable config (or text) file that can be copied or downloaded
+
+### Installation feedback form [#installation-feedback]
+
+Stay tuned!
+
+### Tab switcher [#tab]
+
+The tab switcher, like collapsers, is a way of presenting sets of content in a single location.
+
+```
+<Tabs>
+	<TabsBar>
+		<TabsBarItem id=""></TabsBarItem>
+		<TabsBarItem id=""></TabsBarItem>
+		<TabsBarItem id=""></TabsBarItem>
+	</TabsBar>
+
+	<TabsPages>
+		<TabsPageItem id="">
+		</TabsPageItem>
+		<TabsPageItem id="">
+		</TabsPageItem>
+		<TabsPageItem id="">
+		</TabsPageItem>
+	</TabsPages>
+</Tabs>
+```
+
+
+### Tile dropdown [#tile-dropdown] 
+
+The tile dropdown is how a reader controls the branching content that's shown on a doc. The dropdown content and structure is controlled via the branching doc's YAML file.
+
+### Numbered steps [#numbered-steps]
+
+Nearly every MDX file in a branching doc is represented by a numbered step. The branching doc's numbered steps are automatically generated via the YAML file.
+
+### Embedded config file [#config]
+
+The config file is added to the `/install/config/agent-config` directory. Specific lines can be targeted in the config file for update by the reader.
+
+## How it works [#how]
+
+The install docs live in the `install` directory. There is a single `.yaml` file that defines how and when specific content is swapped in and out of the doc. The specific pieces of content are represented as `.mdx` files.
+
+## Directory overview [#install-directory]
+
+The install framework content lives in `src/install`. 
+
+To create a new install doc, add a directory to `install`.
+
+Your directory structure needs to look like this:
+
+```
+install
```

Comment:
@paperclypse could you clarify that writers need to create a parent directory for their work under "install"? For example, they would create "Java" or "Python" directories.